python/ray/dag/compiled_dag_node.py

```diff
@@ -183,7 +182,7 @@ def do_profile_tasks(
     """
     try:
         for task in tasks:
-            task.prepare()
+            task.prepare(overlap_gpu_communication=overlap_gpu_communication)
```

Comment:
Maybe add a comment to remind others.

Reply:
I mentioned this in the [issue](https://github.com/ray-project/ray/issues/48747#issuecomment-2480474180) I opened. Should be fixed soon.